insights/specs/default.py

```diff
@@ -475,6 +475,7 @@ def md_device_list(broker):
     mount = simple_command("/bin/mount")
     mounts = simple_file("/proc/mounts")
     mssql_conf = simple_file("/var/opt/mssql/mssql.conf")
+    mssql_api_assessment = simple_file("/var/opt/mssql/log/assessments/assessment-latest")
```

Comment:
@wushiqinlou - Sorry for missing this one. Since the core collection is able to collect the file without the uploader.json, please raise another PR to remove this line. It can be added after the collection gets determined and the spec gets approved.